fitbenchmarking/fitbenchmarking
# 445 why docs

#### Description of Work

Fixes #445 


#### Testing Instructions

1. Read and check the output in the relevant section of the docs

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
